lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -555,10 +555,16 @@ def get_address(options)
         options[:billing_address] || options[:address]
       end
 
+      def add_token_transaction_type(xml, options)
```

Comment:
Is it necessary to use options?
Looks like you are using by default 'UT' as a TokenTxnType.